src/Mod/Draft/DraftEdit.py

```diff
@@ -302,14 +324,17 @@ def startEditing(self, event):
         pos = event.getPosition()
         node = self.getEditNode(pos)
         ep = self.getEditNodeIndex(node)
-        if ep is None: return
+        if ep is None:
+            return
 
         doc = FreeCAD.getDocument(str(node.documentName.getValue()))
         self.obj = doc.getObject(str(node.objectName.getValue()))
-        if self.obj is None: return
+        if self.obj is None:
+            return
         self.setPlacement(self.obj)
 
-        FreeCAD.Console.PrintMessage(str(self.obj.Name)+str(": editing node: n° ")+str(ep)+"\n")
+        FreeCAD.Console.PrintMessage(str(self.obj.Name) + str(
+            ": editing node: n° ") + str(ep) + "\n")
```

Comment:
You can break here at the operator, as it's already inside a pair of parentheses.
```py
FreeCAD.Console.PrintMessage(str(self.obj.Name)
                             + str(": editing node: n° ")
                             + str(ep) + "\n")
```